all.jsonnet

```diff
@@ -149,10 +147,79 @@ local qf = t.queryFrontend(commonConfig {
   },
 });
 
+
+local hashrings = import 'kube-thanos/kube-thanos-receive-hashrings.libsonnet';
```

Comment:
That should be necessary, when adding this to the thanos.libsonnet, like every other component. :)